playwright/test/works.test.ts

```diff
@@ -46,7 +46,7 @@ describe('works', () => {
     expect(value).toBe(expectedValue);
   });
 
-  fit('Search by term, filter results, check results, view result', async () => {
```

Comment:
What was `fit` @ishouty ?

Reply:
ive removed this. this was for debugging purposes. slipped in.

allows to skip other tests and focus on this one.

Reply:
[Answer my own question](https://jestjs.io/docs/en/api#testonlyname-fn-timeout)